website/docs/cloud-docs/run/states.mdx

```diff
@@ -136,7 +136,27 @@ _Leaving this stage:_
 
 [permissions-citation]: #intentionally-unused---keep-for-maintainers
 
-## 8. The Apply Stage
+## 8. The Pre-Apply Stage
+
+The pre-apply phase only occurs if there are enabled [run tasks](/cloud-docs/workspaces/settings/run-tasks) in the workspace that are configured to begin before Terraform creates the apply. Terraform Cloud sends information about the run to the configured external system and waits for a `passed` or `failed` response to determine whether the run can continue. The information sent to the external system includes the configuration version of the run.
+
+-> **Note:** Pre-apply run tasks are in beta.
+
+Only runs that are confirmed can enter this phase.
```

Comment:
```suggestion
Only confirmed runs can enter this phase.
```

Suggest changing "runs that are confirmed" to "confirmed runs" to avoid passive voice style issues.